From 78fb0f312b9dc907c3f3c745c22541d4962a4de6 Mon Sep 17 00:00:00 2001
From: Patrick McCarty <patrick.mccarty@intel.com>
Date: Wed, 11 Jul 2018 19:58:25 -0700
Subject: [PATCH 1/5] Import configparser module directly

iniparse.configparser is only present in Fedora's patched version of
iniparse, but it is essentially a wrapper module to import either
"ConfigParser" or "configparser" for Python 2 or 3, respectively.

We only need the Python 3 module, so just import "configparser"
directly.

Signed-off-by: Patrick McCarty <patrick.mccarty@intel.com>
---
 dnf/plugin.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/dnf/plugin.py b/dnf/plugin.py
index 71805f7..64b8931 100644
--- a/dnf/plugin.py
+++ b/dnf/plugin.py
@@ -22,11 +22,11 @@ from __future__ import absolute_import
 from __future__ import print_function
 from __future__ import unicode_literals
 
+import configparser
 import fnmatch
 import glob
 import importlib
 import iniparse.compat
-import iniparse.configparser
 import inspect
 import logging
 import operator
@@ -57,7 +57,7 @@ class Plugin(object):
         files = ['%s/%s.conf' % (path, name) for path in conf.pluginconfpath]
         try:
             parser.read(files)
-        except iniparse.configparser.ParsingError as e:
+        except configparser.ParsingError as e:
             raise dnf.exceptions.ConfigError(_("Parsing file failed: %s") % e)
         return parser
 
-- 
2.19.1

